### DISCUSSION TITLE
LOL智能小助手

### DISCUSSION THREAD

**Discussion by 2001wjh** [InternLM/Tutorial#369] [project]
# 项目目标

创造一个精通LoL相关知识，并能以固定范式在每场对局开始前给出英雄、符文和出装建议的AI小助手。

# 技术路线

#### 2. 搭建向量知识库

​搜集与LoL相关的网络文本（makedown格式和txt格式），使用Langchain框架搭建向量知识库，通过提供检索问答链对象来实现对于 RAG 全流程的封装。

#### 3.微调模型

​基于op.gg网站上高质量的最新英雄数据（已整理好），利用Xtuner或Huggingface框架下的LORA方法进行进行模型微调，让模型在特定prompt下的回复具有绝对正确性。

#### 4. 对改进后的模型进行量化部署

​基于Lmdeploy的量化和部署。（TurboMind推理 + 网页Demo演示 + Python代码集成）

#### 5. 利用Gradio搭建对话的前端页面，发布在Openxlab和Huggingface上。

# 团队成员（排名按入队前后顺序）

2001wjh

ZMcc18

**Comment by biscuit279** [comment]
这个好！做咋样了兄弟们，我也可以来帮帮忙
